README.md

```diff
@@ -53,11 +53,11 @@ Find your preferred SDK in [our list of official SDKs](#unleash-sdks) and import
 If you use the docker compose file from the previous step, here's the configuration details you'll need to get going:
 
 - For front-end SDKs, use:
-  - URL: `http://localhost:3000/proxy`
-  - `clientKey`: `proxy-client-key`
+  - URL: `http://localhost:4242/api/frontend/`
+  - `clientKey`: `<frontend key you create>`
 - For server-side SDKs, use:
   - Unleash API URL: `http://localhost:4242/api/`
-  - API token: `default:development.unleash-insecure-api-token`
```

Comment:
should we keep the init key for simplicity?

Reply:
Again, yeah, I think that's the way to go. That way, people can just go and connect directly instead of having to fiddle in the UI first.